frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -24,13 +24,42 @@ import AssociatedLandForm from '../SidebarContents/AssociatedLandForm';
 import { toast } from 'react-toastify';
 import _ from 'lodash';
 import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import GenericModal from 'components/common/GenericModal';
+import { FaCheckCircle } from 'react-icons/fa';
+import styled from 'styled-components';
+import { IFinancialYear } from 'features/properties/components/forms/subforms/EvaluationForm';
 
 interface IMapSideBarContainerProps {
   refreshParcels: Function;
   properties: IProperty[];
   movingPinNameSpaceProp?: string;
 }
 
+export interface IFormBuilding extends IBuilding {
+  financials: IFinancialYear[];
+}
+export interface IFormParcel extends IParcel {
+  financials: IFinancialYear[];
+  buildings: IFormBuilding[];
+}
+
+const FloatCheck = styled(FaCheckCircle)`
+  margin: 1em;
+  color: #2e8540;
```

Comment:
could you change this to $completed-color?

Reply:
I can't actually (at least as far as I know) use css vars with styled components. If this is something you want to try and figure out I'd love to learn how.